.github/workflows/integration.yml

```diff
@@ -168,6 +168,9 @@ jobs:
           BIGQUERY_TEST_SERVICE_ACCOUNT_JSON: ${{ secrets.BIGQUERY_TEST_SERVICE_ACCOUNT_JSON }}
           BIGQUERY_TEST_ALT_DATABASE: ${{ secrets.BIGQUERY_TEST_ALT_DATABASE }}
           BIGQUERY_TEST_NO_ACCESS_DATABASE: ${{ secrets.BIGQUERY_TEST_NO_ACCESS_DATABASE }}
+          DBT_TEST_USER_1: dbt_test_user_1
+          DBT_TEST_USER_2: dbt_test_user_2
+          DBT_TEST_USER_3: dbt_test_user_3
```

Comment:
These actually need to be:
- `user:buildbot@dbtlabs.com`
- `user:buildbot@fishtownanalytics.com`
- a third one that we need to find / decide - maybe `group:dev-core@dbtlabs.com` ?

Reply:
Done